app/views/users/index.xlsx.axlsx

```diff
@@ -0,0 +1,18 @@
+wb = xlsx_package.workbook
+
+wb.styles do |s|
+  title_style = s.add_style :sz => 20, :alignment => { :horizontal=> :center }, :border => { :style => :thick, :color => "000000", :edges => [:left, :right, :top, :bottom] }
+  wrap_text = s.add_style({:alignment => {:horizontal => :center, :vertical => :center, :wrap_text => true}})
+  border_text = s.add_style(:border => { :style => :thin, :color => "000000", :edges => [:left, :right, :top, :bottom] })
```

Comment:
You should using new syntax for symbol.
For example:
`sz: 20, alignment: {horizontal: :center}`

Reply:
done